astropy/utils/modal_utils.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+__all__ = 'make_modal'
+
+try:
+    import tkinter
+    from tkinter import ttk
+except ImportError:
+    tkinter = None
+
+if tkinter is None:
+    def make_modal(text, title, blink_ms, geometry, font, center_window=True):
+        print(text)
```

Comment:
Maybe use this?

https://github.com/astropy/astropy/blob/64fe8fa3b15089caac67c1806f3ce648bdb73791/astropy/utils/console.py#L297